packages/quantic/force-app/main/default/lwc/quanticSvg/quanticSvg.js

```diff
@@ -0,0 +1,16 @@
+import { LightningElement, api } from 'lwc';
+// @ts-ignore
+import pageNotFound from './pageNotFound.html';
+// @ts-ignore
```

Comment:
these ts-ignore comments are necessary? Just making sure :)

Reply:
my VS code doesn't like it (the import), is this a recurrent problem?

Reply:
if running `tsc` doesn't raise any errors then I would remove it. But it's possible that your vscode is alerting you because the typecheck would indeed fail.

Reply:
Yea importing a template has got to be something SF only understands?